$MirOS: ports/comms/linphone/patches/patch-mediastreamer_osscard_c,v 1.1.7.1 2005/03/18 15:43:50 tg Exp $
--- mediastreamer/osscard.c.orig	Fri Oct  1 23:51:49 2004
+++ mediastreamer/osscard.c	Fri Oct  1 23:52:44 2004
@@ -23,8 +23,13 @@
 #include "msossread.h"
 #include "msosswrite.h"
 
+#if defined(HAVE_SYS_SOUNDCARD_H) || \
+    (defined(HAVE_SOUNDCARD_H) && (defined(__NetBSD__) || defined(__OpenBSD__)))
 #ifdef HAVE_SYS_SOUNDCARD_H
 #include <sys/soundcard.h>
+#else
+#include <soundcard.h>
+#endif
 
 #include <errno.h>
 #include <fcntl.h>
